Error when compiling to ARM7

When compiling to android arm7 on a linux host I get the following error : 
```
blas_server.c: In function 'blas_thread_init':
blas_server.c:575:16: error: storage size of 'rlim' isn't known
  struct rlimit rlim;
                ^
blas_server.c:578:9: warning: implicit declaration of function 'getrlimit' [-Wimplicit-function-declaration]
         if(0 == getrlimit(RLIMIT_NPROC, &rlim)) {
         ^
blas_server.c:578:27: error: 'RLIMIT_NPROC' undeclared (first use in this function)
         if(0 == getrlimit(RLIMIT_NPROC, &rlim)) {
                           ^
blas_server.c:578:27: note: each undeclared identifier is reported only once for each function it appears in
blas_server.c:575:16: warning: unused variable 'rlim' [-Wunused-variable]
  struct rlimit rlim;
                ^
make[1]: *** [blas_server.o] Erreur 1
```
Here's how I start the build : 
```
make HOSTCC=gcc CC=arm-linux-androideabi-gcc NO_LAPACK=1 TARGET=ARMV7
```

I managed to correct the error modifying the file `driver/others/blas_server.c` from this : 
```
#if defined(OS_LINUX) || defined(OS_NETBSD) || defined(OS_DARWIN)
#include <dlfcn.h>
#include <signal.h>
#include <sys/time.h>
#include <sys/resource.h>
#endif
```
To this : 
```
#if defined(OS_LINUX) || defined(OS_NETBSD) || defined(OS_DARWIN)
#include <dlfcn.h>
#include <signal.h>
#include <sys/time.h>
#endif
#include <sys/resource.h>
```

It works but it obviously broke everything on other platforms ! Is there something as a OS_ARM define ?

--- a/driver/others/blas_server.c
+++ b/driver/others/blas_server.c
@@ -70,7 +70,7 @@ USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 /*********************************************************************/
 
 #include "common.h"
-#if defined(OS_LINUX) || defined(OS_NETBSD) || defined(OS_DARWIN)
+#if defined(OS_LINUX) || defined(OS_NETBSD) || defined(OS_DARWIN) || defined(OS_ANDROID)
 #include <dlfcn.h>
 #include <signal.h>
 #include <sys/resource.h>

